Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -2571,24 +2579,29 @@ def handle_user_unassignment(update_args):
         update_args['assigned_user_pretty_name'] = None
 
 
-def handle_outgoing_issue_closure(update_args, inc_status):
-    if inc_status == 2:
-        if MIRROR_IN_CLOSE_REASON not in update_args.get('closeNotes', ''):
-            update_args['resolve_comment'] = update_args.get('closeNotes', '')
-            update_args['status'] = XSOAR_RESOLVED_STATUS.get(update_args.get('closeReason', 'Other'))
-            demisto.debug(f"Closing Remote incident with status {update_args['status']}")
-        else:
-            demisto.debug('Ignore closing Remote incident because incident closed during mirroring in')
+def handle_outgoing_issue_closure(remote_args):
+    update_args = remote_args.delta
+    current_remote_status = remote_args.data.get('status')
```

Comment:
```suggestion
    current_remote_status = remote_args.data.get('status') if remote_args.data else None
```
Check that `remote_args.data` isn't None.